### PR TITLE
[Mellanox] Add support of SN5600 platform on top of Nvidia ASIC simulation

### PR DESCRIPTION
**Pull request by raphaelt-nvidia**
Signed-off-by: Raphael Tryster <raphaelt@nvidia.com>

#### Why I did it

Add new Spectrum-4 system support SN5600 on top of Nvidia ASIC simulator.

#### How I did it

Add all relevant system and simulator SKU.
Updated syseeprom.hex and related directories to reflect Nvidia SN5600 brand name.

#### How to verify it

Tested init flow, basic show commands, up interfaces, traffic test.

#### Description for the changelog

Support Nvidia SN5600 simulation.

### CODE REVIEW

**Review by liat-grozovik — APPROVED** [review]

File: device/mellanox/x86_64-nvidia_sn5600_simx-r0/ACS-SN5600/buffers_defaults_t0.j2 [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{#
+    Copyright (c) 2021 NVIDIA CORPORATION & AFFILIATES.
```
Comment (liat-grozovik) [inline device/mellanox/x86_64-nvidia_sn5600_simx-r0/ACS-SN5600/buffers_defaults_t0.j2:2]: @raphaelt-nvidia please change to 2022 as we didn't merge on 2021 :-)

**Comment by raphaelt-nvidia** [comment]
This PR was created for 202111 at Judy Josephs' request, with identical content having been pushed to master.  It is now no longer needed, as Judy already merged to 202111 on 9th January 2022.  The commit is https://github.com/Azure/sonic-buildimage/commit/dd8fd57bb543f5229cf021fcf1813f9a021ab814.

**Pull request closed without merging**